kdl_parser/package.xml

```diff
@@ -27,11 +27,12 @@
   <build_depend>roscpp</build_depend>
```

Comment:
I don't think this improves anything so no need to make a change here, but I think `roscpp` could also be only a `<test_depend>` if the CMakeLists.txt was updated. The only ros related header I see included outside of a test is https://github.com/ros/kdl_parser/blob/71e728edffa4d41cfbd3b44839a3bfd7d04aa368/kdl_parser/src/kdl_parser.cpp#L41

which comes from rosconsole 
https://github.com/ros/ros_comm/blob/lunar-devel/tools/rosconsole/include/ros/console.h